configs/sst_virtualization-all.yaml

```
@@ -82,3 +82,15 @@ data:
           description: Add support for Windows guests to virt-v2v and virt-p2v
           dependencies:
             - libguestfs
+    # Package in RHEL and Fedora is different
+    - srpm_name: trustee-guest-components
+      build_dependencies:
+        - rust-toolset
+        - openssl-devel
+        - tss2-devel
+        - tpm2-tss-devel
+      limit_arches:
+        - x86_64
+      rpms:
+        - rpm_name: trustee-guest-components
+          description: Tools that run in confidential VMs, attest and get secrets from Trustee
```

Based on the build deps, I'd guess there will be some runtime deps too, along the lines of:
```suggestion
          description: Tools that run in confidential VMs, attest and get secrets from Trustee
          dependencies:
            - openssl-libs
            - tss2
            - tpm2-tss
```

---

@uril Is this correct? I don't see runtime deps in the spec that you provided.

---

I think all the runtime deps are automatically added when the package is built

---

You can manually add deps:
  openssl-libs
  tpm2-tss

---

Not to the spec file (as library dependencies are autogenerated), but they would be needed in the placeholder.  That being said, see my response to the other thread.